tests/stacks/generic/test_data/expected/apps_v1_deployment_tf-job-operator.yaml

```diff
@@ -38,6 +38,6 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        image: gcr.io/kubeflow-images-public/tf_operator:vmaster-gd455e6ef
+        image: gcr.io/kubeflow-images-public/tf_operator:vmaster-ga2ae7bff
```

Comment:
seems `tests/stacks/gcp/test_data/expected/apps_v1_deployment_tf-job-operator.yaml` was not updated.

Reply:
Thanks @Jeffwan!
@jlewi Do you know why CD didn't update that part of stacks tests?